content/post/2019-12-19-mutable-api.Rmarkdown

```diff
@@ -0,0 +1,184 @@
+---
+title: Objects that feel mutable in R
+date: '2019-12-26'
+slug: mutable-api
+tags:
+  - package development
+---
+
+In R, most often, to change an object, you need to re-assign its new value to it. But sometimes, things feel different because objects are mutable or it seems they are, be it in base R code or in the code of packages. Why and how provide a mutable API/interface in R code? In this blog post, we shall explore <s>few odd examples</s> a few examples to better understand mutable APIs in R.
```

Comment:
Why and how _to_ provide?

_a_ few odd examples? Or just put the `a` before the crossed out part?